test/Api.IntegrationTest/AdminConsole/Controllers/OrganizationUsersControllerTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Net;
+using System.Net.Http.Headers;
+using Bit.Api.IntegrationTest.Factories;
+using Bit.Seeder.Recipes;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Bit.Api.IntegrationTest.AdminConsole.Controllers;
+
+public class OrganizationUsersControllerTest : IClassFixture<ApiApplicationFactory>, IAsyncLifetime
+{
+    private readonly HttpClient _client;
+    private readonly ApiApplicationFactory _factory;
+    private readonly ITestOutputHelper _testOutputHelper;
+
+    public OrganizationUsersControllerTest(ApiApplicationFactory factory, ITestOutputHelper testOutputHelper)
+    {
+        _factory = factory;
+        _testOutputHelper = testOutputHelper;
+        _client = _factory.CreateClient();
+    }
+
+    public Task InitializeAsync()
+    {
+        return Task.CompletedTask;
+    }
+
+    public Task DisposeAsync()
+    {
+        _client.Dispose();
+        return Task.CompletedTask;
+    }
+
+    [Fact(Skip = "Performance testing")]
+    public async Task Get_SmallOrg()
+    {
+        var db = _factory.GetDatabaseContext();
+        var seeder = new OrganizationWithUsersRecipe(db);
+
+        var orgId = seeder.Seed("Org", 100, "large.test");
```

Comment:
Should this be `small.test`?

On that note - can we combine these 2 tests and just use inline theory data to seed different numbers and different domains?

Reply:
😞

Reply:
Keeping this unresolved as I hid other feedback here:

> On that note - can we combine these 2 tests and just use inline theory data to seed different numbers and different domains?